README.md

```diff
@@ -1,2 +1,50 @@
 # Platalea
 Understanding visually grounded spoken language via multi-tasking
+
+![install and run in conda env](https://github.com/egpbos/platalea/workflows/install%20and%20run%20in%20conda%20env/badge.svg?branch=master)
+[![codecov](https://codecov.io/gh/spokenlanguage/platalea/branch/master/graph/badge.svg)](https://codecov.io/gh/spokenlanguage/platalea)
+
+
+
+## Install
+
+```sh
+conda create -n platalea python==3.8 pytorch -c conda-forge -c pytorch
+conda activate platalea
+pip install torchvision
+pip install .
+```
```

Comment:
This is equivalent, `pip install .` also uses the information in setup.py. The advantage is that it's also properly registered in pip, which may be important for dependencies. And it's easier to uninstall, since setup.py uses `easy_install` for installing, which often doesn't clean up well, whereas with pip you don't have any such problems. Don't think it matters much in our case, but in general, nowadays `pip install .` seems to be preferred over `python setup.py install`.

Reply:
Great!